### DISCUSSION TITLE
Upgraded to symfony/flex v1.19.2 but still flex.symfony.com not found

### DISCUSSION THREAD

**Discussion by meekruu** [symfony/symfony#46679] [Q&A]
Hello,

I am starting to work again on a 4.4 symfony project and have updated symfony flex according to https://symfony.com/blog/upgrade-flex-on-your-symfony-projects. When I run the project on my development machine, everything is fine. But the `composer install` command in the GitLab pipeline still fails with 
```
$ composer install --no-scripts
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
The following exception probably indicates you have misconfigured DNS resolver(s)
                                                                                                                   
  [Composer\Downloader\TransportException]                                                                         
  curl error 6 while downloading https://flex.symfony.com/versions.json: Could not resolve host: flex.symfony.com
```

Does anyone have an idea, why this happens? I already disabled caching the vendor folder...

**Comment by lyrixx** [comment]
It looks like your `composer.lock` is not commited.

**Reply by meekruu** [reply]
It is:
[
![image](https://user-images.githubusercontent.com/107549539/173824260-d869d888-b95e-4dec-a705-402d04eaa0c0.png)
](url)

**Comment by meekruu** [comment]
What about the allow-plugins part in composer.json?
```
        "allow-plugins": {
            "symfony/flex": true
        }
```
Is this still needed? When I run composer install --no-plugins, it works...

**Reply by stof** [reply]
Well, if you disable the plugin, Flex won't be applied, and so you won't have the features of Flex.

**Reply by meekruu** [reply]
Of course I would still like to have it. But then I assume, it would be alright to run the composer install without the plugin on the GitLab pipeline, wouldn't it?

**Comment by stof** [comment]
@meekruu does your gitlab setup use a global install of the Flex plugin ?

**Reply by meekruu** [reply]
I don't think so. I tried to check with composer `global show`, but this says, there is no composer.json in /root/.composer.
